fluss-server/src/main/java/org/apache/fluss/server/replica/ReplicaManager.java

```diff
@@ -450,14 +451,15 @@ public void appendRecordsToLog(
             int timeoutMs,
             int requiredAcks,
             Map<TableBucket, MemoryLogRecords> entriesPerBucket,
+            UserContext userContext,
```

Comment:
Add `@Nullable` to the parameter if it may pass `null` parameter (even if only tests pass `null`). Same to other methods.